source/megengine.js

```diff
@@ -3,30 +3,96 @@
 
 var megengine = megengine || {};
 var base = base || require('./base');
+var zip = zip || require('./zip');
 
```

Comment:
Import `flatbuffers` dependency and test this works for Electron app, browser and Python server.

Reply:
done

Reply:
Remove the `zip` import as well.

Reply:
done